src/styles/base/_typography.scss

```diff
@@ -21,7 +21,7 @@ html {
 
 body {
   @include font-styles(body);
-  background-color: var(--color-fill);
+  background-color: var(--color-text-background);
```

Comment:
Why is this change necessary?

Reply:
It's slightly more correct in that we want the text and text-background colors as the default body color for the color/background-color respectively. Technically in swift-docc-render, these are the same (fill is used for text-background), but other applications extending this may choose to have different semantic colors for the same thing.